pkgs/by-name/so/soundconverter/package.nix

```diff
@@ -0,0 +1,117 @@
+{
+  lib,
+  fetchurl,
+  # Optional due to unfree license.
+  faacSupport ? false,
+  glib,
+  python3Packages,
+  gtk3,
+  wrapGAppsHook3,
+  gsettings-desktop-schemas,
+  intltool,
+  xvfb-run,
+  gobject-introspection,
+  gst_all_1,
+  fdk-aac-encoder,
+}:
+let
+
+  pythonWrapPackages = [
```

Comment:
This is how `dependencies` work. Can't understand why you have to use this dirty hack with `buildPythonApplication`.

Reply:
See https://nixos.org/manual/nixpkgs/unstable/#reference